apps/condo/domains/miniapp/schema/B2BAppAccessRightSet.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * Generated by `createschema miniapp.B2BAppAccessRightSet 'app:Relationship:B2BApp:CASCADE;'`
+ */
+
+const { historical, versioned, uuided, tracked, softDeleted, dvAndSender } = require('@open-condo/keystone/plugins')
+const { GQLListSchema } = require('@open-condo/keystone/schema')
+
+const access = require('@condo/domains/miniapp/access/B2BAppAccessRightSet')
+
+
+const B2BAppAccessRightSet = new GQLListSchema('B2BAppAccessRightSet', {
+    schemaDoc: 'A set of access rights for a service user integrated with B2BApp.',
+    fields: {
+
+        app: {
+            schemaDoc: 'Link to B2BApp',
+            type: 'Relationship',
+            ref: 'B2BApp',
+            isRequired: true,
+            knexOptions: { isNotNullable: true }, // Required relationship only!
+            kmigratorOptions: { null: false, on_delete: 'models.CASCADE' },
+            access: {
+                update: false,
+            },
+        },
+
```

Comment:
Write a note, that a lot of fields will be added with preprocessor/plugin. Not obvious, where to find them :)